insights/terraform/projects/ab2d/main.tf

```diff
@@ -25,6 +25,14 @@ module "bucket" {
   ]
 }
 
+data "aws_s3_bucket" "moderate_bucket" {
+  bucket = "bfd-insights-moderate-577373831711"
```

Comment:
Suggest replacing this with the caller identity data source

Reply:
Good point. Pushed a new change.